Signal/src/ViewControllers/MediaGalleryViewController.swift

```diff
@@ -437,7 +450,7 @@ class MediaGallery: NSObject, MediaGalleryDataSource, MediaTileViewControllerDel
 
                             // fade out content behind the pageViewController
                             // and behind the presentation view
-                            self.navigationController.view.backgroundColor = Theme.backgroundColor
+                            self.navigationController.view.backgroundColor = Theme.galleryBackgroundColor
```

Comment:
Huh, so these "gallery..." properties are essentially "always dark theme"?  I wonder if we should call them "dark..." instead so that we can reuse them in places like the image picker, etc.

Reply:
Ok, I'll rename.